From d8e741222d7c1b158460dc225dbb064a77f490bf Mon Sep 17 00:00:00 2001
From: weideng <wei.a.deng@intel.com>
Date: Fri, 21 Apr 2017 00:52:03 +0000
Subject: [PATCH 3589/4301] Change Trusty Kconfig to build for X86 Arch only

commit c96bed370b743c84b535b81c7840d2d0154fb73a from
git://git@github.com:01org/linux-intel-4.9.git

Currently Trusty only works on x86, so the module should never build
for other archs except x86. Add this patch to add 'depends' part on
drivers/trusty/Kconfig to disable them.

Signed-off-by: weideng <wei.a.deng@intel.com>
---
 drivers/trusty/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/trusty/Kconfig b/drivers/trusty/Kconfig
index 0b6b88e..44d0219 100644
--- a/drivers/trusty/Kconfig
+++ b/drivers/trusty/Kconfig
@@ -6,6 +6,7 @@ menu "Trusty"
 
 config TRUSTY
 	tristate "Trusty"
+	depends on X86
 	default n
 
 config TRUSTY_FIQ
-- 
1.7.5.4

